drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ActionInsertFactColumnSynchronizer.java

```diff
@@ -35,6 +35,14 @@ public int priority() {
     }
 
     @Override
+    public boolean handlesAppend( final MetaData metaData ) throws ModelSynchronizer.MoveColumnVetoException {
+        if ( !( metaData instanceof ColumnMetaData ) ) {
+            return false;
+        }
+        return ( (ColumnMetaData) metaData ).getColumn() instanceof ActionInsertFactCol52;
+    }
+
+    @Override
```

Comment:
@manstis All the new handlesAppend methods will be always the same as the handlesUpdate methods? Maybe a method extraction makes sense here.